mainnet-1/config/pre-genesis.json

```diff
@@ -48,11 +48,12 @@
             ],
             "delegated_free": [],
             "delegated_vesting": [],
+            "start_time": "1656316800",
```

Comment:
You need to pull this up one level.
Outside of "base_vesting_account" and on the same level as "@type"